backoffice/templates/backoffice/user.html

```diff
@@ -131,68 +130,31 @@
             {% endif %}
             </button>
         {% endif %}
-
-    </div>
-
-</div>
-
-
-<!-- Modal -->
-<form method="POST" method=".">{% csrf_token %}
-<input type="hidden" name="action" value="ban-user">
-<input type="hidden" name="value" value="{% if disabled %}reenable{% else %}disable{% endif %}">
-<div class="modal fade" id="ban-user" tabindex="-1" role="dialog" aria-labelledby="ban-user-label" aria-hidden="true">
-    <div class="modal-dialog">
-        <div class="modal-content">
-            <div class="modal-header">
-                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                    <span aria-hidden="true">&times;</span>
-                </button>
-                <h4 class="modal-title" id="ban-user-label">{% trans "Please confirm" %}</h4>
-            </div>
-            <div class="modal-body">
-
-                <h3>{% if disabled %}{% trans "Confirm unban of user" %}{% else %}{% trans "Confirm ban of user" %}{% endif %}</h3>
-                {% trans "Full name" context "user-full-name" %}: <strong>{{ userform.instance.profile.name }}</strong><br>
-                {% trans "Username" %}: <strong>{{ userform.instance.username }}</strong>
-                <br><br>
-            </div>
-            <div class="modal-footer">
-                <button type="button" class="btn btn-default" data-dismiss="modal">{% trans "Cancel" %}</button>
-                <button type="submit" class="btn btn-primary">{% trans "Confirm" %}</button>
-            </div>
-        </div>
+            
+        <h3>{% trans "Certificates" %}</h3>
+        <table class="table table-condensed">
+            <tr>
+                <th>{% trans "Course" %}</th>
+                <th>{% trans "Grade" %} </th>
+                <th>{% trans "Status" %}</th>
+                <th>{% trans "Pdf" %}</th>
+            </tr>
+            {% for certificate in certificates %}
+                <tr>
+                    <td>{{ certificate.course_id }}</td>
+                    <td> {{ certificate.grade }} <span id='change-grade-button' data-course-id="{{certificate.course_id}}" class='glyphicon glyphicon-pencil' data-toggle="modal" data-target="#change-grade"></span> </td>
```

Comment:
<a target="_blank" href="{% url 'user_profile' key userform.instance.id %}">
                    <span title="{% trans 'See user contributions to course forum' %}"class="glyphicon glyphicon-eye-open" aria-hidden="true"></span>
                </a>